target/linux/mediatek/dts/mt7981b-snr-cpe-ax2.dts

```diff
@@ -0,0 +1,333 @@
+// SPDX-License-Identifier: GPL-2.0-only OR MIT
+
+/dts-v1/;
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+#include "mt7981.dtsi"
+
+/ {
+	model = "SNR CPE-AX2";
+	compatible = "snr,cpe-ax2", "mediatek,mt7981";
+
+	aliases {
+		label-mac-device = &gmac0;
+
+		led-boot = &ledgear_blue;
+		led-failsafe = &ledgear_red;
+		led-running = &ledgear_blue;
+		led-upgrade = &ledgear_red;
+
+		serial0 = &uart0;
+	};
+
+	chosen {
+		rootdisk = <&ubi_fit_volume>;
+		stdout-path = "serial0:115200n8";
+                bootargs-append = " root=/dev/fit0 rootwait";
+        };
+
+	gpio-keys {
+		compatible = "gpio-keys";
+
+		button-0 {
+			label = "mesh";
+			gpios = <&pio 0 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RFKILL>;
+			debounce-interval = <60>;
+		};
+
+		button-1 {
+			label = "reset";
+			gpios = <&pio 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_wan_red: led-0 {
+			label = "WAN_RED";
+			color = <LED_COLOR_ID_RED>;
+			gpios = <&pio 4 GPIO_ACTIVE_LOW>;
+		};
+
+		ledgear_red: led-1 {
+			label = "LED_GEAR_RED";
+			color = <LED_COLOR_ID_RED>;
+			gpios = <&pio 5 GPIO_ACTIVE_LOW>;
+		};
+
+		led_2g: led-2 {
+			label = "LED_2G_BLUE";

```

Comment:
`color`/`function` properties exist and this `label` property should be removed.

Reply:
The priority of `label` is higher than `color`/`function` properties and it will be used for LED names even if `color`/`function` properties exist.